vignettes/common.R

```diff
@@ -41,8 +41,16 @@ dir_tree_hipercow <- function(path) {
   withr::with_dir(path, fs::dir_tree(glob = "hipercow/*", invert = TRUE))
 }
 
+sys_getenv <- function(x) {
+  res <- Sys.getenv(x)
+  if (res == "") {
+    return(NULL)
+  }
```

Comment:
```suggestion
  res <- Sys.getenv(x, NA_character_)
  if (is.na(res)) {
    return(NULL)
  }
```

this is how I usually do this, I think?

Reply:
Right - I was close!